src/gs-updates-page.ui

```diff
@@ -11,6 +11,39 @@
       <object class="GtkBox">
         <property name="visible">True</property>
         <property name="orientation">vertical</property>
+	<style>
```

Comment:
I know this looks misaligned but it's because I have made sure to use space indentation as that's what should be used in UI files (even if **some** pre-existing bits are using tabs).